src/api/GetAgent.ts

```diff
@@ -0,0 +1,130 @@
+/**
```

Comment:
Eventually, this and the POC need to agree.   I think this needs to happen before this PR gets merged.

Reply:
True, although based on my initial read this looks like it agrees with the WG's doc